pkg/plugins/trivy/plugin.go

```diff
@@ -794,7 +794,9 @@ func (p *plugin) getPodSpecForClientServerMode(ctx trivyoperator.PluginContext,
 	var containers []corev1.Container
 
 	trivyConfigName := trivyoperator.GetPluginConfigMapName(Plugin)
-
+	volumeMounts = append(volumeMounts, getScanResultVolumeMount())
```

Comment:
are all the containers using the same volume mounts?

Reply:
yes , probably we will have to set a different scan/.result_<num>.json name for each